### DISCUSSION TITLE
Pixi add: Option to Review Implied Changes Before Adding Packages.

### DISCUSSION THREAD

**Discussion by gitkief** [prefix-dev/pixi#2552] [Ideas]
When adding a package to an environment using `pixi add`, it seems there is currently no way to review and confirm the implied changes to the environment and lock file beforehand (apologies if this option already exists and I’ve overlooked it). Would it be a good idea to add an option to review the implied changes, similar to the behavior of `conda install`?

**Comment by ruben-arts** [comment]
Correct we don't do this, we always write to the lockfile so that could give you the diff if you git commited in between. But I could see this feature be useful too! We're open to PR's!

**Comment by sabetts** [comment]
I'm trying to install a very specific version of llama.cpp that will match the old version of CUDA I have installed. With conda/mamba I can issue an install command and review the versions to confirm that it's not installing the CPU build nor is it installing a build that uses the wrong version of CUDA. Considering the dependencies are several gigabytes, for speed and peace of mind it would be very helpful to have a --confirm or --review option.
